kustomize/install/default/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+namespace: postgres-operator
+
+commonLabels:
+  postgres-operator.crunchydata.com/control-plane: postgres-operator
+  app.kubernetes.io/name: pgo
+  app.kubernetes.io/version: 5.1.0
+
+bases:
+- ../crd
+- ../rbac/cluster
+- ../manager
+
+images:
+- name: postgres-operator
+  newName: registry.developers.crunchydata.com/crunchydata/postgres-operator
+  newTag: ubi8-5.1.0-0
+
+patchesJson6902:
```

Comment:
Just use `patches`

Reply:
`patchesJson6902` is the older (deprecated?) field from before `patches` existed.